src/analysis/special_functions/improper_integrals.lean

```diff
@@ -0,0 +1,107 @@
+/-
+Copyright (c) 2023 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+import measure_theory.integral.integral_eq_improper
+import measure_theory.integral.exp_decay
+import analysis.special_functions.integrals
+
+/-!
+# Evaluation of specific improper integrals
+
+This file contains some integrability results, and evaluations of integrals, over `ℝ` or over
+half-infinite intervals in `ℝ`.
+
+## See also
+
+- `analysis.special_functions.integrals` -- integrals over finite intervals
+- `analysis.special_functions.gaussian` -- integral of `exp (-x ^ 2)`
+-/
+
+open real set filter measure_theory
+open_locale topology
+
+lemma integral_exp_neg_Ioi : ∫ (x : ℝ) in Ioi 0, exp (-x) = 1 :=
+begin
+  refine tendsto_nhds_unique (interval_integral_tendsto_integral_Ioi _ _ tendsto_id) _,
+  { simpa only [neg_mul, one_mul] using exp_neg_integrable_on_Ioi 0 zero_lt_one, },
+  { simpa only [interval_integral.integral_comp_neg, neg_zero, integral_exp, exp_zero,
+      tsub_zero] using tendsto.const_sub 1 tendsto_exp_neg_at_top_nhds_0, },
+end
+
+/-- If `0 < c`, then `(λ t : ℝ, t ^ a)` is integrable on `(c, ∞)` for all `a < -1`. -/
+lemma integrable_on_Ioi_rpow_of_lt {a : ℝ} (ha : a < -1) {c : ℝ} (hc : 0 < c) :
+  integrable_on (λ t : ℝ, t ^ a) (Ioi c) :=
+begin
+  refine integrable_on_Ioi_of_interval_integral_norm_bounded
```

Comment:
This looks more painful than it should be. I have developed some API for this kind of things in #18844, do you think it would help here (and elsewhere in this PR)? (see for instance `integrable_on_Ioi_deriv_of_nonneg'` there).

Reply:
#18844 has been merged. Can you merge master and use it here?

Reply:
I made a shot at using #18844 here, but it actually doesn't seem to simplify matters all that much. E.g. if we want to integrate `x ^ a` for `a < -1` using the new API from #18444, then we need to either go through the proof that the derivative of `x ^ (a + 1) / (a + 1)` is `x ^ a`, which takes a few lines, or bypass this by quoting FTC-1, but then we are basically back at the original statement of `interval_integral_tendsto_integral_Ioi`.